src/.editorconfig

```
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.{yml,yaml}]
+indent_size = 2
+
+[docker-compose.yml]
+indent_size = 4
```

I think we use `indent_size = 2` in all our Vue based applications
```suggestion
[*.{yml,yaml}]
indent_size = 2

[*.{js,ts,vue,css}]
indent_size = 2

[docker-compose.yml]
indent_size = 4
```